dataone-node/src/main/java/org/irods/jargon/dataone/MemberNodeService.java

```diff
@@ -251,9 +263,12 @@ public MNChecksum handleGetChecksum(@PathParam("id") final String pid,
 	@GET
 	@Path("/replica/{id}")
 	@Produces(MediaType.APPLICATION_OCTET_STREAM)
-	public void handleReplica(@PathParam("id") final String pid, @Context final HttpServletResponse response)
+	public void handleReplica(@PathParam("id") final String pid, @Context final HttpServletRequest request,
+							  @Context final HttpServletResponse response)
 			throws InvalidToken, ServiceFailure, NotAuthorized, NotFound, NotImplemented, InsufficientResources {
 
+		getAuthChecker().checkAuthorization(request);
+
```

Comment:
I've only added the certificate validation on the `/log` and `/replica/{id}` endpoints so far. I may add it to additional endpoints in the future if necessary.